### PR TITLE
Migrate to Compose Multiplatform

### PR DESCRIPTION
**Pull request by thelumiereguy**
### Changes 
-  Moved Composables to common Compose module - which will be shared across platforms

### Additions
- Added Entry points for Desktop, iOS and Web. 
- Added 3 fields to Jekyll documentation - for thumbnail and, Source URL for Compose and Processing.
- Upgraded Compose and Kotlin versions

### CODE REVIEW

File: compose_common/src/commonMain/kotlin/dev/thelumiereguy/creative_coding_compose/CreativeCodingExampleScreen.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+@Composable
+fun CreativeCodingExampleScreen(
+    modifier: Modifier
+) {
+    CreativeCodingComposeTheme {
+        BasicFlowField(
+            modifier,
+        )
+    }
+}
```
Comment (CuriousNikhil) [inline compose_common/src/commonMain/kotlin/dev/thelumiereguy/creative_coding_compose/CreativeCodingExampleScreen.kt:17]: [s}: We should add ktlint to the format code.

Comment (thelumiereguy) [reply compose_common/src/commonMain/kotlin/dev/thelumiereguy/creative_coding_compose/CreativeCodingExampleScreen.kt:17]: +1 

I'll do that yes!

File: compose_common/src/commonMain/kotlin/dev/thelumiereguy/creative_coding_compose/examples/basic_flow_field/BasicFlowField.kt [hunk 2]
```diff
@@ -112,12 +118,10 @@ fun BasicFlowField(modifier: Modifier = Modifier) {
 
                 points = points.map { point ->
 
-                    val frameNoise = noise1D(frame.toDouble())
-
                     val angle = map(
                         noise2D(
-                            frameNoise.toFloat() * point.x.toDouble(),
-                            frameNoise * point.y.toDouble()
+                            frame.toDouble().toFloat() * point.x.toDouble(),
```
Comment (CuriousNikhil) [inline compose_common/src/commonMain/kotlin/dev/thelumiereguy/creative_coding_compose/examples/basic_flow_field/BasicFlowField.kt]: `toDouble().toFloat()` ?

Comment (thelumiereguy) [reply compose_common/src/commonMain/kotlin/dev/thelumiereguy/creative_coding_compose/examples/basic_flow_field/BasicFlowField.kt]: Let me check this!

**Review by CuriousNikhil — APPROVED** [review]

**Pull request merged**